test/libsinsp_e2e/event_capture.cpp

```diff
@@ -79,7 +79,6 @@ void event_capture::capture()
 				m_capture_started = true;
 				m_condition_started.notify_one();
 			}
-			delete m_inspector;
```

Comment:
No need to `delete` unique ptr anymore.
Also, `sinsp` dtor autoamtically calls stop capture and `scap_close` for us.